### PR TITLE
Implement POST request for portfolioStep function

### PR DESCRIPTION
**Pull request by zachclark-ccpo**
Adds the capability to save the user input from the portfolio step (step 1) of the portfolio draft wizard.
Implements `POST /portfolioDrafts/{portfolioDraftId}/portfolio`

Ticket: AT-6368

### CODE REVIEW

File: provisioning/src/handlers/portfolioDrafts/portfolio/post.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  const ps: PortfolioStep = {
+    name: req.body.name,
+    description: req.body.description,
+    dod_components: req.body.dod_components,
+    portfolio_managers: req.body.portfolio_managers
+  }
+
+  const pf: PortfolioFull = {
+    ...resource,
+    updated_at: now,
```
Comment (jeffsegal-ccpo) [inline provisioning/src/handlers/portfolioDrafts/portfolio/post.ts:34]: This is absolutely fine for now, but I'm hoping that we can eventually define some cross-cutting capabilities to automatically handle these types of things (e.g. updating audit fields).

File: provisioning/src/handlers/portfolioDrafts/portfolio/post.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import dayjs from 'dayjs'
+import { PortfolioStep } from '../../../lib/models/PortfolioStep'
+import { PortfolioFull } from '../../../lib/models/PortfolioFull'
+import { CosmosClient } from '@azure/cosmos'
+
+export async function handler (context: Context, req: HttpRequest): Promise<void> {
+  const connectionString = process.env['COSMOS-CONNECTION-STRING']
+  context.log('Connecting to ' + connectionString)
+  const client = new CosmosClient(connectionString)
+  const id: string = req.params.portfolioDraftId
```
Comment (jeffsegal-ccpo) [inline provisioning/src/handlers/portfolioDrafts/portfolio/post.ts:11]: Glad to see straightforward support for named path parameters here.

**Review by jeffsegal-ccpo — APPROVED** [review]

File: provisioning/src/handlers/portfolioDrafts/portfolio/post.ts (hunk 2 above)
Comment (zachclark-ccpo) [reply provisioning/src/handlers/portfolioDrafts/portfolio/post.ts:11]: Explicitly defining it the serverless.yml was giving me problems, its nice that Azure assumes the path param automagically though.

**Review by jasonburkert-ccpo — APPROVED** [review]
Peer reviewed.  Looks good.

**Pull request merged**